README.md

```diff
@@ -352,3 +352,66 @@ Planned features:
   * Support for integer textures and arithmetic
 
   * Compute shaders
+
+
+Halide for Hexagon HVX
+======================
+Halide now supports the Qualcomm Hexagon DSP. The Hexagon DSP provides a set of 64 and 128 byte vector instructions - the Hexagon Vector eXtensions (HVX). HVX is well suited to image processing and Halide for Hexagon HVX will generate the appropriate HVX vector instructions from a program authored in Halide.
+
+For Halide programmers, a 'hexagon' schedule has been added which informs Halide to generate a binary suitable for execution with the Hexagon DSP Simulator, or on target using a Qualcomm Snapdragon 820 device.
+In order to build and run a Halide application for the Hexagon HVX processor:
+  1. Obtain and build LLVM and clang v3.9 or later from llvm.org
+  2. Download and install the Hexagon SDK and version 8.0 Hexagon Tools
+  3. Build and run an example for Hexagon HVX
+
+#### 1.       Obtain and build LLVM and clang v3.9 or later from llvm.org
+LLVM 3.9 is currently under development. These are the same instructions as above for building Clang/LLVM, but for trunk Clang/LLVM instead of 3.7.
+
+    cd <path to llvm>
+    svn co http://llvm.org/svn/llvm-project/llvm/trunk .
+    svn co http://llvm.org/svn/llvm-project/cfe/trunk ./tools/clang
+    mkdir build
+    cd build
+    cmake -DLLVM_ENABLE_TERMINFO=OFF -DLLVM_TARGETS_TO_BUILD="X86;ARM;NVPTX;AArch64;Mips;PowerPC;Hexagon" -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release ..
+    make -j8
+    export LLVM_CONFIG=<path to llvm>/build/bin/llvm-config
+    export CLANG=<path to llvm>/build/bin/clang
+
+#### 2.       Download and install the Hexagon SDK and version 8.0 Hexagon Tools
+Go to https://developer.qualcomm.com/software/hexagon-dsp-sdk/tools
+1)      Select the Hexagon Series 600 Software and download the 3.0 version for Linux.
```

Comment:
The formatting of this section got really butchered. I would replicate the formatting from the numbered list from lines 363-365